src/main/resources/assets/twilightforest/lang/zh_tw.lang

```diff
@@ -626,3 +675,26 @@ modifier.stalwart.name=偉岸
 modifier.stalwart.desc=§o勇敢和大膽。§r\n擊中一個實體可以讓你感到底氣。
 modifier.veiled.name=含蓄
 modifier.veiled.desc=§o像幽靈一樣！§r\n拋射物將隱形。
+
+# Immersive Engineering
+item.tfEngineeringShader=着色器：%s
```

Comment:
The "shader" here is this thing: https://github.com/BluSunrize/ImmersiveEngineering/blob/76f3a2bb4a30ebe1d2da31c0b26f90d67e131729/src/main/resources/assets/immersiveengineering/lang/en_us.lang#L413